src/distributed/topology_local_wrapper.ts

```diff
@@ -11,11 +11,10 @@ import * as serialize_error from "serialize-error"
 export class TopologyLocalWrapper {
 
     private uuid: string;
-    private process: NodeJS.Process;
+    private process: any; // NodeJS.Process
```

Comment:
write longer comment, explain that methods are missing

Reply:
fixed